setup.py

```diff
@@ -57,12 +57,13 @@ def get_version(file="flake8_isort.py"):
     zip_safe=False,
     install_requires=[
         'flake8 >= 3.2.1, <4',
-        'isort[pyproject] >= 4.3.5, <5',
+        'isort >= 4.3.5, <6',
```

Comment:
Update lower isort version to 4.3.21 as it is the lastest 4.x version in pypi: https://pypi.org/project/isort/#history

Reply:
`4.3.21 >= 4.3.5` is `True`, so no problem here.

Reply:
you're right, my bad